frontend/taipy-gui/src/components/Taipy/DateRange.tsx

```diff
@@ -218,20 +226,23 @@ const DateRange = (props: DateRangeProps) => {
                                 format={props.format}
                                 id={id && id + "-field"}
                                 className={getSuffixedClassNames(className, "-text")}
-                                width={props.width && "100%"}
+                                width={props.width}
+                                disabled
+                                label={props.labelStart}
```

Comment:
I don't understand those changes.

- Why change the width ?
- Field has no `disabled` or `label` property